website/content/development/gpio.md

```diff
@@ -85,3 +84,101 @@ At this point, we should be able to see the high/low signal on the multimeter,
 alternating between 3.3V (high) and 0V (low) every 5 seconds:
 
 <a href="/img/2020-06-15-gpio.jpg"><img src="/img/2020-06-15-gpio.thumb.jpg" srcset="/img/2020-06-15-gpio.thumb.2x.jpg 2x,/img/2020-06-15-gpio.thumb.3x.jpg 3x" width="700" style="border: 1px solid grey; margin-bottom: 2em; margin-top: 1em"></a>
+
+## Setting up an input pin
+
+To setup a Pin as an input and to act on change to High level, such as a button,
+we must poll the Pin periodically. This is needed because the
```

Comment:
How about:

…we must poll the Pin periodically because periph.io’s edge detection, which examples like the [periph.io/device/button example](https://periph.io/device/button/) rely on, is [not available on gokrazy](https://github.com/gokrazy/gokrazy/issues/233).

Reply:
Sounds better, thanks 😊